Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -3281,6 +3285,10 @@ void finishMolProcessing(RWMol *res, bool chiralityPossible, bool sanitize,
     }
     MolOps::assignStereochemistry(*res, true, true, true);
   } else {
+    if (!Chirality::getUseLegacyStereoPerception()) {
```

Comment:
Why is this only done when we're using the new stereo perception code? Is bad stereo not a problem with the legacy code?
Some comment here would be nice

Reply:
The legacy code does its own removal of stereo info when 2 groups are the same, so calling Chirality::removeBadStereo is not needed in legacy mode.  I added a comment to the code there